src/App.tsx

```diff
@@ -1,29 +1,89 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { PaginationInfo } from './components/PaginationInfo';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const items: number[] = getNumbers(1, 42).map(n => n);
+
+const perPageOptions = [3, 5, 10, 20].map((el, i) => {
+  return { id: i, val: el };
+});
```

Comment:
There is no need to use the index `i` as the id for `perPageOptions`. If the values are unique, they can be used as keys directly in React components.